content/en/docs/languages/erlang/propagation.md

```diff
@@ -4,26 +4,28 @@ weight: 60
 cSpell:ignore: elli
 ---
 
-## Cross Service Propagators
+{{% docs/languages/propagation erlang %}}
+
+## Automatic context propagation
 
 Distributed traces extend beyond a single service, meaning some context must be
 propagated across services to create the parent-child relationship between
 Spans. This requires cross service
 [_context propagation_](/docs/specs/otel/overview/#context-propagation), a
 mechanism where identifiers for a Trace are sent to remote processes.
 
-Instrumentation Libraries for HTTP frameworks and servers like
+Instrumentation libraries for HTTP frameworks and servers like
 [Phoenix](https://github.com/open-telemetry/opentelemetry-erlang-contrib/tree/main/instrumentation/opentelemetry_phoenix),
 [Cowboy](https://github.com/open-telemetry/opentelemetry-erlang-contrib/tree/main/instrumentation/opentelemetry_cowboy),
 [Elli](https://github.com/open-telemetry/opentelemetry-erlang-contrib/tree/main/instrumentation/opentelemetry_elli)
 and clients like
 [Tesla](https://github.com/open-telemetry/opentelemetry-erlang-contrib/tree/main/instrumentation/opentelemetry_tesla)
-will automatically inject or extract context using the globally registered
+automatically inject or extract context using the globally registered
 propagators. By default the global propagators used are the W3C
 [Trace Context](https://w3c.github.io/trace-context/) and
 [Baggage](https://www.w3.org/TR/baggage/) formats.
 
-These global propagators can be configured by the Application environment
+You can configure global propagators using the the application environment
```

Comment:
I do the capitalization of Application on purpose when referring to an OTP Application and not the generic term "application". I guess its not an official OTP convention so I'm fine either way but this will make it out of sync with the rest of the otel docs where I should be capitalizing as well.

Other option would be to say "OTP application environment".

Reply:
I'll leave it capitalized for now then! Thanks!

Reply:
I think we should make sure that we use "OTP application environment" where possible, otherwise it is going to be confusing, since our docs span across different languages, but that might be a separate issue

Reply:
I replaced all instances. @tsloughter WDYT? https://github.com/open-telemetry/opentelemetry.io/pull/4801/commits/1739d680cc9dd7e25b3737a1b61f57d60197c38b